frontend/src/components/core/Container.tsx

```diff
@@ -12,22 +12,32 @@ const createContainer = (Container: any, props: MyProps) => {
 
 export const FlexContainer = styled.div`
   display: flex;
-  flex-wrap: wrap;
-  max-height: 100vh;
-  overflow: hidden;
+  flex: auto;
+  flex-direction: column;
+  margin: 15px;
 `;
 
 export const FlexInfoBar = styled.div`
-  flex: 1 0 100%;
   padding: 0.5rem;
   height: 15px;
   line-height: 15px;
   text-align: center;
 `;
 
-export const FlexPanel = styled.div`
-  flex: 1;
-  padding: 1rem;
+export const Panel = styled.div`
+  height: 100%;
+  padding: 15px;
```

Comment:
Thoughts on changing to `rem` to match the rest of the site? (Note: I don't think this applies to border-width or radius.)